optuna/integration/fastai.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import print_function
+
+import optuna
+from optuna import type_checking
+
+if type_checking.TYPE_CHECKING:
+    from typing import Any  # NOQA
+
+try:
+    from fastai.basic_train import Learner  # NOQA
+    from fastai.callbacks import TrackerCallback
+    _available = True
+except ImportError as e:
+    _import_error = e
+    _available = False
+    TrackerCallback = object
+
+
+class FastAIPruningCallback(TrackerCallback):
+    """FastAI callback to prune unpromising trials for fastai.
+
+    .. note::
+        This callback is for fastai :math:`\\le` 2.0.
```

Comment:
I think that `le` stands for "less than or equal" (see: https://medemanabu.net/latex/operators/).
Is this correct? (i.e., does this callback support fastai version 2.0?)

Reply:
Right, it should be just ”less than”.

In fact, fastai v2 is in alpha and I don't check whether this work on fastai v2 yet.

Reply:
I see. Thanks.

Reply:
Quick note

As I installed and skimmed the coming fastai implementation of `TrackerCallback` (https://github.com/fastai/fastai_dev/blob/master/dev/17_callback_tracker.ipynb), it seems at this moment, the library is installed as `fastai2` and there'll be some changes.
So, if the latest fastai is public, the integration callback should be implemented in a separate file.